### ISSUE TITLE
change color is not visible upon hover mouse on "WATCH VIDEO"

### ISSUE THREAD

**Issue by coderkid19** [topcoder-platform/qa-fun#289]
Steps :
1) Go to https://www.topcoder.com/solutions/
2) Hover mouse on "WATCH VIDEO"
3) Observe for any color change

Expected Result:
Button color should change also color contrast should be such that it is visible with background color.

Actual color:
Color change upon hover mouse on "WATCH VIDEO", is not visible.

Screencast 

[4.zip](https://github.com/topcoder-platform/qa-fun/files/4576025/4.zip)

Device/OS/Browser Information:
Windows 10 
Chrome Version 81.0.4044.129 (Official Build) (64-bit)

**Comment by coderkid19** [comment]
![BrowserInfo_Chrome_81 0 4044 113](https://user-images.githubusercontent.com/51490878/80992243-dcbe1d80-8e56-11ea-86c9-591b85d0c3c7.png)
![DeviceInfo_Windows10](https://user-images.githubusercontent.com/51490878/80992252-dd56b400-8e56-11ea-815a-62806541fcee.png)
